GoogleDAIDemo/app/src/main/java/com/jwplayer/googledai/MainActivity.java

```diff
@@ -0,0 +1,91 @@
+package com.jwplayer.googledai;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.longtailvideo.jwplayer.JWPlayerView;
+import com.longtailvideo.jwplayer.configuration.PlayerConfig;
+import com.longtailvideo.jwplayer.events.AdErrorEvent;
+import com.longtailvideo.jwplayer.events.listeners.AdvertisingEvents;
+import com.longtailvideo.jwplayer.media.ads.dai.ImaDaiSettings;
+import com.longtailvideo.jwplayer.media.playlists.PlaylistItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity
+        implements AdvertisingEvents.OnAdErrorListener{
+
+    JWPlayerView mPlayerView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        mPlayerView = findViewById(R.id.jwplayer);
+
+        List<PlaylistItem> playlist = new ArrayList<>();
+        String videoId = "bbb-clear";
+        String cmsId = "2474148";
+        ImaDaiSettings.StreamType streamType = ImaDaiSettings.StreamType.DASH;
+        ImaDaiSettings imaDaiSettings = new ImaDaiSettings(videoId, cmsId, streamType, null);
+        String fallbackUrl = "http://playertest.longtailvideo.com/adaptive/bipbop/gear4/prog_index.m3u8";
+
+        PlaylistItem playlistItem = new PlaylistItem.Builder()
```

Comment:
I think the overall code looks good. 
Since this is user facing I think we need some comments about where values like videoId, cmsId, come from, when you use which stream type, amd what use cases the fallbackUrl kicks in.